### ISSUE TITLE
Color themes support

### ISSUE THREAD

**Issue by igorshubovych** [tldr-pages/tldr-node-client#40]
- [ ] Implement several themes that should look good on either black or white terminal.
- [ ] Themes should be just preset colors.
- [ ] Current theme can be set in config file.
- [ ] The default theme should be colorless.

See discussion in #39

**Comment by rprieto** [comment]
Thanks for opening this! As discussed in #18, should themes include how tokens are displayed? E.g. wrapped in colours, or wrapped in a special "token" character, etc...

**Comment by sbrl** [comment]
I like the idea of storing themes under `~/.tldr/themes`. As for specifying own settings, I think that point (2) covers everything - to create your own theme all you have to do it copy an existing file and alter it.

Here's what I imagine a theme file to look like:

```
{
"wrapTokens": true,             // wrap tokens like     head -c {{count}}
  "ansiColors": true,             // if false, output is just raw text
  "ansiPalette": {
    "text": "green",
    "command-background": "black",
    "command-foreground": "red",
    "command-token": "white"
  }
}
```

I don't think it needs to contain the theme name as that information is already contained within the filename.

**Comment by agnivade** [comment]
Fixed by #117
